examples/cpp/HelloWorldExample/Monitor.cpp

```diff
@@ -97,9 +104,49 @@ void Monitor::run()
         std::cout << std::endl;
         get_fastdds_latency_mean();
         get_publication_throughput_mean();
+
+        // TODO: decide whether inactive entities must be inside json bump
+        // If not so, reset must be done before bumping file
+        if (!bump_file_.empty())
```

Comment:
I think it makes sense to dump the inactive entities before removing them.

```suggestion
        if (!bump_file_.empty())
```